docs/README.md

```diff
@@ -28,6 +28,8 @@ As a *developer*, designate the appropriate workload type to execute your compon
 #### [Create Component from Scratch](how-to/create_component_from_scratch.md)
 Build a component from source code to use for testing.
 
+#### [Using Helm/Kustomize to manage OAM yamls](how-to/using_helm_kustomize_manage_oam.md)
```

Comment:
nits: need a one-line description like the others above and below:

![image](https://user-images.githubusercontent.com/1701782/67971407-f477c900-fbc9-11e9-9a84-797057851207.png)